### DISCUSSION TITLE
Did you like the game? 🙂

### DISCUSSION THREAD

**Discussion by vlad-reshetylo** [vlad-reshetylo/theunclickable#1] [Polls]
Feel free to share your experience, suggestions, ideas, criticism, etc

**Comment by dannypernik** [comment]
I'm just glad it's over 🤣 

(But to be clear, I very much enjoyed it and shared it with a friend.)

**Reply by HouQiming** [reply]
Cheated a few levels and wrote an automation script:

```javascript
{var ee={isTrusted:true};ee.__proto__=MouseEvent.prototype;var btn=document.querySelectorAll('button')[0];if(btn){btn.textContent='Click me';btn.style.backgroundColor='#00ff00';btn.onclick(ee);}else{theUnclickable.click();}}
```

**Comment by SargoDarya** [comment]
Was a fun little thing however there was an endless loop at some point where it was constantly trying to load google fonts when dev tools are opened.

**Reply by vlad-reshetylo** [reply]
Hmm, that's weird. Do you know how to reproduce that behaviour?

**Comment by AutoPotato** [comment]
It was very fun! And I learned a fair amount of things as a web dev noob :D
When first time playing it was quite confusing for me tho, maybe explaining where you can look and where you can't would be a good idea?
For example, I assume you shouldn't look into the JS file, or adding a button yourself (I tried to do that in level 8 and 10).

**Reply by vlad-reshetylo** [reply]
> It was very fun! And I learned a fair amount of things
Thank you! I'm glad that it was not just fun, but useful 🙂

**Comment by InomjonIsmaylov** [comment]
The idea is fantastic! I enjoyed the game and I am looking forward to the next levels!

**Comment by EnjSadman** [comment]
lvl 11 was the hardest. Great idea, great realization

**Comment by VladBogach** [comment]
Можна трошки задач по джсу додати, + в сорсах лежав код джсний, так було легко дізнатись умову виконання. Забагато Css. Задача з зміною тексту трішки неочевидно, бо треба замінити саме на "Click me", варіанти "Do not touch me" => "Touch me"|"Do touch me"... всі не спрацюють, як на мене дарма, бо так забавніше)

Дуже сильно не вистачає рестарту лвла, тому що я не 1 раз видаляв необхідний мені елемент з DOM

**Reply by vlad-reshetylo** [reply]
До речі про рестарт левела мені прям багато людей писали) Думаю, додам

Щодо задач: це, наприклад, які? нічого не приходить на думку, якщо чесно

**Comment by spirit-q2** [comment]
nice and cute! 🤗
for the 9th and 10th, I had to look into the sources 😊 🫢 but since I'm a BE developer I consider it as a "fair cheating" 😄

**Comment by iN1kk0** [comment]
Лайтовий, але файний пет-проєкт. Гра для тих, хто ще не втомився від DevTools після робочого дня 🫠

**Reply by vlad-reshetylo** [reply]
Так і задумувалось :)

**Comment by aleksanderbabayev** [comment]
I have managed to pass two levels by pressing tab key and then enter. Not sure that is was expected completion :)

**Comment by AnastasiiaHombalevska** [comment]
Я лише починаю свою "подорож" світом розробки й дуже страждаю від думки "нічо не знаю". Але ця гра і про доказ, що це не зовсім так =) хах
Цікаво, змушує спробувати щось не очевидне!
Приєднуюсь до пропозиції вище щодо рестарту рівня. А про левел Do not touch me - сподобався!

ПС. навчилась лазити в сорс😄
Дякую!

**Reply by vlad-reshetylo** [reply]
Дякую за відгук, радий що сподобалось 🙂
